src/merge_tools.rs

```diff
@@ -102,13 +101,10 @@ pub enum ConflictResolveError {
         adds: usize,
     },
     #[error(
-        "The output file is either unchanged or empty after the editor quit.\n Details: tool \
-         '{tool_binary}' was executed with args {args:?}."
+        "The output file is either unchanged or empty after the editor quit (run with --verbose \
+         to see the exact invocation)."
     )]
-    EmptyOrUnchanged {
-        tool_binary: String,
-        args: Vec<String>,
-    },
```

Comment:
It's probably better to squash commits since the previous commit is almost reverted.

Reply:
I didn't want to lose it until I was sure we aren't doing it. It's a bit annoying to keep updating the one test that doesn't use `insta`.

Reply:
Maybe you can `duplicate` the previous commit locally? If we decided to not use tracing here, we could backout this PR and reapply your original commit.

Reply:
OK.